power app sample/readme.md

```diff
@@ -0,0 +1,76 @@
+![banner](https://i.ibb.co/QQCVq4m/banner.png)
+
+## Overview
+
+This is a fully packaged solution for the **D365 Fraud Protection Portal Admin App** that serves multiple purposes:
+
+- Demonstrate the extensibility capabilities of Fraud Protection Event Tracing and provide an open-source reference for those creating their own custom Power App.
+- Provide IT Engineers and Fraud Managers with a toolset for actively monitoring latency measurements for their Fraud Protection instance as well as maintaining an interactive audit log of Fraud Protection portal related events.
+
+The app will work directly with Fraud Protection **Audit Events** and **Latency Events** data.
+
+## Prerequisites
+
+- Azure subscription
+- Event Hub Namespace
+- Two Event Hubs
+- Dynamics 365 Fraud Protection event tracing subscriptions connected to your two Event Hubs (one for **Audit Events**, one for **Latency Events**)
+- Power Apps subscription
+
+For help setting up the above, check the documentation in **Reference Links** in the following section.
+
+## Reference Links
+
+[Dynamics 365 Fraud Protection Event Tracing Documentation](https://docs.microsoft.com/en-us/dynamics365/fraud-protection/event-tracing)
+
+## Import Solution
+
+1. Download and save the `D365FraudProtectionPortalAdminPowerAppSolution.zip` file in this repository to your computer.
+2. In the [Power Apps Portal](https://make.preview.powerapps.com/), navigate to the **Solutions** page in the left sidebar.
+3. Click **Import**, and in the window that is opened, click **Choose File** and load the `D365FraudProtectionPortalAdminPowerAppSolution.zip` file.
+4. Click **Next**. Then, click **Import**.
```

Comment:
I got an error that mentioned I don't have enough permissions:
![image](https://user-images.githubusercontent.com/165767/91143428-5e84c180-e667-11ea-8249-9ddca5710e4c.png)

Your PR description alluded to this possibly happening since I don't have CDS access.  Can you include a note/alert near the top of these instructions - like you did for the templates - spelling out what access you need?  Or, put it in the pre-reqs area if that works better?  But, somewhere early on so folks don't spend too much time if there are other things they need to do first.

Reply:
Okay. It's been added as a pre-req.

Reply:
Thanks